src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -4482,7 +4521,7 @@ INSERT INTO tab_calculation_method_rates VALUES('VascularEndoVolume','VOLUME_Vas
 INSERT INTO tab_calculation_method_rates VALUES('VascularEndoVolume','VOLUME_VascularEndo_tes','SA_pls_int*d_endo','Volume');
 INSERT INTO tab_calculation_method_rates VALUES('VascularEndoVolume','VOLUME_VascularEndo_uil','SA_pls_int*d_endo','Volume');
 INSERT INTO tab_calculation_method_rates VALUES('VascularEndoVolume','VOLUME_VascularEndo_uje','SA_pls_int*d_endo','Volume');
-INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_EffectiveLumenConcentration','EnableSupersaturarion ? M/Liquid : min(Solubility/MW ; M/Liquid)','Concentration (molar)');
+INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_EffectiveLumenConcentration','Liquid>0 ? (EnableSupersaturarion ? M/Liquid : min(Solubility/MW ; M/Liquid)) : 0','Concentration (molar)');

```

Comment:
Changed the formula so that it does not return NaN if `Liquid `is zero.
Otherwise it crashed with cattle, because the Gi-TRact is de-facto disabled there (all volumes are set to 0), and our new parameter generation in core did not create this parameter in the simulation at all, because it was NaN